admin/nodes/views.py

```diff
@@ -467,3 +467,25 @@ def post(self, request, *args, **kwargs):
                                     ' if the problem persists get a developer to fix it.')
 
         return redirect(reverse_node(self.kwargs.get('guid')))
+
+
+class RemoveStuckRegistrationsView(PermissionRequiredMixin, TemplateView):
```

Comment:
You could probably inherit from the `RestartStuckRegistrationsView`, and then just override the template_name, and the `post` method.